acme/acme/__init__.py

```diff
@@ -10,3 +10,12 @@
   https://github.com/ietf-wg-acme/acme/tree/draft-ietf-acme-acme-01
 
 """
+import sys
+import warnings
+
+if sys.version_info[:2] == (3, 3):
+    warnings.warn(
+            "Python 3.3 support will be dropped in the next release of"
+            "Certbot. Please upgrade your Python version.",
```

Comment:
We need a space between "of" and "Certbot".

Reply:
Also, we should say "acme" here as there are other ACME clients using this module.